compressed_image_transport/include/compressed_image_transport/compressed_publisher.h

```diff
@@ -66,7 +66,7 @@ class CompressedPublisher : public image_transport::SimplePublisherPlugin<Compre
                const PublishFn& publish_fn) const;
 
   struct Config {
-    // Compression format to use "jpeg" or "png"
+    // Compression format to use "jpeg", "png" or "tiff".
```

Comment:
would be good to update https://github.com/ros2/common_interfaces/blob/master/sensor_msgs/msg/CompressedImage.msg#L12
as well

Reply:
Sounds good, I've opened https://github.com/ros2/common_interfaces/pull/154.